### DISCUSSION TITLE
Donation via bank transfer possible?

### DISCUSSION THREAD

**Discussion by ghost** [evilsocket/opensnitch#534] [General]
Hello, I would like to donate some money to you and to know if you accept donations via bank transfer. I know that there is the possibility to donate as a patreon, but I don't want to create more accounts and share my data, if it's not necessary.

**Comment by ghost** [comment]
I think there should also be an option to pay with PayPal, cryptocurrencies, liberapay, opencollective, overall I think it's a good idea to give as many options as possible.

**Comment by gustavo-iniguez-goya** [comment]
So finally I opened a liberapay account: https://liberapay.com/gus/ as well as github's sponsor page: https://github.com/sponsors/gustavo-iniguez-goya
